tests/flow/test_graph_info.py

```diff
@@ -89,6 +89,15 @@ def consumeStream(self, stream, drop=True):
         # convert stream events to LoggedQueries
         logged_queries = [LoggedQuery(e[1]) for e in events]
 
+        # wait until `n_items` events are consumed
+        elapsed = 10
+        while len(logged_queries) < n_items and elapsed > 0:
+            time.sleep(0.2)
+            elapsed -= 0.2
+
+            events = self.conn.xrevrange(stream, '+', '-')
+            logged_queries = [LoggedQuery(e[1]) for e in events]
```

Comment:
this assignment is wrong, it overwrites `logged_queries` I think each iteration of this loop should add elements to `logged_queries`.